edx_name_affirmation/services.py

```diff
@@ -0,0 +1,40 @@
+"""
+Name Affirmation Service
+"""
+
+import types
+
+
+class NameAffirmationService:
```

Comment:
This references the same pattern found in https://github.com/edx/edx-proctoring/blob/master/edx_proctoring/services.py